src/main/java/ladder/domain/WinningItems.java

```diff
@@ -0,0 +1,39 @@
+package ladder.domain;
+
+import ladder.exception.WinningItemsException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class WinningItems {
+    private final List<WinningItem> winningItems;
+
+    public WinningItems(String[] itemsName) {
+        validateWinningItems(itemsName);
+
+        List<WinningItem> winningItems = new ArrayList<>();
+
+        for (String item : itemsName) {
+            WinningItem winningItem = new WinningItem(item);
+            winningItems.add(winningItem);
+        }
+        this.winningItems = winningItems;
+    }
+
+    private void validateWinningItems(String[] itemsName) {
+        for (String item : itemsName) {
+            if (item == null) {
+                throw new WinningItemsException();
+            }
+        }
+    }
```

Comment:
미션이 stream의 사용이니 이 부분도 stream을 사용해보면 어떨까요?
여러 방법이 있겠지만 아래와같이도 될거같아요.
```suggestion
    private void validateWinningItems(String[] itemsName {
    Arrays.stream(itemsName).filter(Objects::isNull).findFirst().ifPresent(WinningItemsException::new);
    }
```

Reply:
이 부분말고 다른 부분들도 stream을 사용할 수 있는 부분이 있는지 한번 쭉 다시 둘러보시면 좋을 것 같아요 :)

Reply:
몇몇 부분 가능해보이는곳은 수정했습니다!. ResultView.printAllResult()와, drawLine()부분도 스트림으로 처리 해보고 싶었는데 하지 못했습니다ㅜ

Reply:
코멘트 남겨두도록 할게요~